adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.h

```diff
@@ -0,0 +1,20 @@
+//
+//  GADPangleRTBNativeRenderer.h
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GADPangleRTBNativeRenderer : NSObject<GADMediationNativeAd>
```

Comment:
As native ad is not currently supported, can you save the code on your end and remove this file?